### PR TITLE
issue #116026

### PR DESCRIPTION
**Pull request by chetanguptaa**
#### What type of PR is this?
issue #116026

/kind bug

#### What this PR does / why we need it:
I added an else if condition that checks if the probe is in an unknown state 
#### Which issue(s) this PR fixes:
issue #116026 

Fixes 
added an else if condition that checks if the probe is in an unknown state
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/prober/prober.go [hunk 1]
```diff
@@ -112,8 +112,9 @@ func (pb *prober) probe(ctx context.Context, probeType probeType, pod *v1.Pod, s
 	if result == probe.Warning {
 		pb.recordContainerEvent(pod, &container, v1.EventTypeWarning, events.ContainerProbeWarning, "%s probe warning: %s", probeType, output)
 		klog.V(3).InfoS("Probe succeeded with a warning", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name, "output", output)
-	} else {
-		klog.V(3).InfoS("Probe succeeded", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
```
Comment (bart0sh) [inline pkg/kubelet/prober/prober.go:116]: @chetanguptaa This info message should be moved down instead of being removed, I believe.

**Comment by bart0sh** [comment]
/priority backlog
/kind cleanup

@chetanguptaa please [sign the CLA](https://github.com/kubernetes/community/blob/master/CLA.md) and rebase to get rid of merge commit, thanks.

File: pkg/kubelet/prober/prober.go (hunk 1 above)
Comment (chetanguptaa) [reply pkg/kubelet/prober/prober.go:116]: oh yes yes surely !!

**Comment by bart0sh** [comment]
@chetanguptaa This PR looks like a duplicate of #117038. Do we really need both?

**Pull request closed without merging**